### ISSUE TITLE
Bad text align

### ISSUE THREAD

**Issue by Cromoray** [zhongyang219/TrafficMonitor#1554]
![Screenshot 2023-03-13 160838](https://user-images.githubusercontent.com/36559019/224743538-cf9178e6-182f-491b-89ec-eef5919162ef.png)

**Comment by f465gt** [comment]
This appeared after the latest Win11 updated. So I assume MS fiddled with the taskbar settings.

**Comment by Shubham123498765** [comment]
Quick fix is to uninstall this update

**Comment by CurtisWong458** [comment]
22621.1413

**Comment by f465gt** [comment]
Hi,
I think it was the KB5022913 update that shows this effect. And you can't uninstall it. Nice.

**Comment by Shubham123498765** [comment]
But it's just a temporary fix only.
Maintainer release a fix.

**Comment by bebablub** [comment]
A (test) release for the majority of us would be great...

**Comment by numbfall** [comment]
Its fixed, use the build [here](https://github.com/zhongyang219/TrafficMonitor/actions/runs/4458814046) and set the desired vertical offset in the settings.

**Comment by f465gt** [comment]
> Its fixed, use the build [here](https://github.com/zhongyang219/TrafficMonitor/actions/runs/4458814046) and set the desired vertical offset in the settings.

It works very nicely! Thanks a lot!

**Comment by Shubham123498765** [comment]
![image](https://user-images.githubusercontent.com/82265932/227137176-3bca2c53-48e1-4e6f-803e-9e5e83def34f.png)
Virus total is flagging this.

**Comment by statiks** [comment]
> Its fixed, use the build [here](https://github.com/zhongyang219/TrafficMonitor/actions/runs/4458814046) and set the desired vertical offset in the settings.

sorry but how do I add this to the exe I'm new to this

**Comment by f465gt** [comment]
Just download the x64 or x86 TrafficMonitor artifact zip file that you find at the "here" link. Stop the TrafficMonitor and unzip all files into the TM folder. Then restart ist.

**Comment by statiks** [comment]
> Just download the x64 or x86 TrafficMonitor artifact zip file that you find at the "here" link. Stop the TrafficMonitor and unzip all files into the TM folder. Then restart ist.

Thanks, links weren't clickable earlier for some reason-

**Comment by EnArvy** [comment]
> Its fixed, use the build [here](https://github.com/zhongyang219/TrafficMonitor/actions/runs/4458814046) and set the desired vertical offset in the settings.

The artifacts have expired, and are no longer available. Please create a new release :(

**Comment by ZhangWebber** [comment]
> Its fixed, use the build [here](https://github.com/zhongyang219/TrafficMonitor/actions/runs/4458814046) and set the desired vertical offset in the settings.

This builds are expired. Can you create new release or artifacts? Thanks

**Comment by bcutter** [comment]
> and set the desired vertical offset in the settings.

@numbfall Where can one set "the vertical offset"? Checked the settings 4 times, nothing there. A screenshot would be very helpful...

**Comment by Shubham123498765** [comment]
#1687 Install new build then you will get vertical offset setting

**Comment by bcutter** [comment]
No, this "new build" get's wiped by my AV software immediately. Listed as "compromised application" !

![grafik](https://github.com/zhongyang219/TrafficMonitor/assets/13799156/f8b59ef5-95bc-4a07-b379-4112eeaf177f)

**Comment by Shubham123498765** [comment]
It's false positive,even if you check for the official build it show the same result.
I am using it and it is running fine.
![image](https://github.com/zhongyang219/TrafficMonitor/assets/82265932/91487b0e-b4b4-4c9c-8749-fdb778d6d3e4)

**Comment by bcutter** [comment]
The official release is working just fine. Just the sideloaded unofficial crap version (which is very likely totally fine but misses probably some signatures or sth. like that) was immediately wiped as shown in my screenshot above. The official version is ignored by the AV software. All those sideloaded unofficial one constantly trigger AV software. What a bummer.

**Comment by andankz** [comment]
Hi the alignment issue shown after updating windows 24h2

**Comment by carlox97** [comment]
![image](https://github.com/user-attachments/assets/32d1f3d0-96cc-423a-96b4-e86e925d6c99)
Is there a way to fix this? "Taskbar window appears to the left of the taskbar" is broken since 24H2
